src/components/right_page/Gallery.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import deptEvent from "../../assets/img/dept_event.png";
+import deptMkt from "../../assets/img/dept_mkt.png";
+import deptPr from "../../assets/img/dept_pr.png";
+import deptTech from "../../assets/img/dept_tech.png";
```

Comment:
I don't think this is the optimal way to import images since in the future we will have many different images.

Reply:
By the way, now you can add some real photos.